configs/ote/custom-object-detection/mobilenetV2_ATSS/model.py

```diff
@@ -75,6 +75,6 @@
 dist_params = dict(backend='nccl')
 log_level = 'INFO'
 work_dir = 'output'
-load_from = None
+load_from = '/usr/src/app/external/mmdetection/ote_data/MODELS/mobilenetV2_ATSS/coco_snapshot.pth'
```

Comment:
What is this?

Reply:
It is the path of the model snapshot in CI.
We did not implement a mechanism that patches/substitutes the file model.py for tests, so wrote the test paths right inside the model.py

Reply:
Bad news is that it'll break everything. You cannot use the model with invalid/unreachable path in `load_from` config parameter. Please come up with a different approach here.

Reply:
Is it possible to push the weights of the model to a server as for other models? It will resolve all the issues.

Another option: we could use environment variables:
```python
import os
model_path = os.environ.get('OTE_TEST_MODEL_PATH')
if model_path is not None:
    load_from = os.path.join(model_path, 'mobilenetV2_ATSS/coco_snapshot.pth')
else:
    load_from = None
```
Note that this will be a temporal decision until the weights are loded to the server.

The third option is to patch the model.py before loading. In this case we should load `OTE_TEST_MODEL_PATH` in test_ote_api.py, create temporary folder, copy all the files from the `template_dir` to the temporary folder, and patch model.py in it. 
But it will break this relative path to `configuration.yaml` in `template.yaml`:
```yaml
hyper_parameters:
  base_path: ../../../../mmdet/apis/ote/apis/detection/configuration.yaml
```
To avoid breaking the relative paths I should 
* either copy the whole git folder to the temporary folder just to patch `model.py` 
* or create the temporary folder just inside the git folder during the test

All these options are a bit weird (except the first one, but it is the longest), so you could propose what should be done here.

Reply:
The option 2 is implemented and it works.